package.json

```diff
@@ -30,7 +30,9 @@
 		"lint": "eslint --ext .js,.vue src",
 		"lint:fix": "eslint --ext .js,.vue src --fix",
 		"stylelint": "stylelint src",
-		"stylelint:fix": "stylelint src --fix"
+		"stylelint:fix": "stylelint src --fix",
+		"test": "node_modules/.bin/mocha --timeout 30000 tests/js/*",
```

Comment:
You should be able to just use `mocha` ?
Any reason you used a relative node_modules path ?

Reply:
You are right! I've just updated the call to mocha.